pynicotine/gtkgui/userbrowse.py

```diff
@@ -856,8 +1129,12 @@ def on_download_files_to(self, *args):
         if not os.path.exists(path) or not os.path.isdir(path):
             path = download_folder
 
+        # ToDo: Make translatable
+        str_title="Select Destination for Downloading File(s) from User"
```

Comment:
There is some KeyError with this which I cannot seem to resolve, it seems choose_dir doesn't like the _() syntax.

Reply:
Variable name conflict on line 1119. Rename `_` to something like `_path_start`.